buildtap.py

```diff
@@ -304,7 +304,7 @@ def package(self):
 
         for f in (os.path.join(self.top, 'COPYING'), os.path.join(self.top, 'COPYRIGHT.GPL')):
             src=open(f, mode='rb')
-            dst.write(src.read()+'\r\n')
+            dst.write(src.read()+'\r\n'.encode())
```

Comment:
1. You can delete the comment above.
2. Please also fix the readme file and update the python section in it.
3. They open it with read binary mode `rb`. Can you open it in a text mode with no need in encoding?
4. How does it effect the `LF -> CRLF` problem? How does `license.txt` looks like on Windows and on Linux? https://stackoverflow.com/a/20653073/241557

Reply:
1. ok.
2. ok. I think this can work on both, python 2.7 and python 3.x. @MaksymSlota can you confirm it's working for you on 2.7?
3. I'll check it.
4. I honestly don't know. I just looked at the `license.txt` on my windows machine it it looked good. Does it matter how it looks on Linux? It can only be installed on windows anyway...

Reply:
Let's try to `cat` it in linux device and see how it looks like.
Also replace `\r\n` with `os.linesep`

Reply:
I changed the read/write of the files to be as text.
I check it on mac and linux machine and it looks fine on both.